src/pe-cases-card/widget/pe-cases-card.u-server-script.js

```diff
@@ -0,0 +1,23 @@
+(function() {
+	/* populate the 'data' object */
+	/* e.g., data.table = $sp.getValue('table'); */
+	
```

Comment:
Please update the table option name and provide a fallback table; in the case a table isn't specified in the options.